server/player/player.go

```diff
@@ -93,8 +93,9 @@ type Player struct {
 }
 
 // New returns a new initialised player. A random UUID is generated for the player, so that it may be
-// identified over network.
-func New(name string, skin skin.Skin, pos mgl64.Vec3) *Player {
+// identified over network. You can either pass on player data you want to load or
+// you can leave the data as nil to use default data.
+func New(name string, skin skin.Skin, pos mgl64.Vec3, data *Data) *Player {
```

Comment:
Now that I think of it, it might be a good idea to just keep the Data out of the New function and only have it in NewWithSession. I can't think of a time where this would be useful for just a normal player entity.

Reply:
That way the thing with having to load the inventory twice is cleaned up as well.